Muthomi254/TravelApp-Backend
# Dan

DESCRIPTION

This PR creates tables and adds data to tables in the db 

How can this be manually tested?

After cloning the repo:-
delete the instance and migrations present

Run/ Bash
$ pip install Faker
$ flask db init
$ flask db migrate -m"initial migration"
$ flask db upgrade
$  python seed.py

Observe the terminal for any errors or validations in the seed.py
Check instance db.sqlite
![Screenshot from 2024-02-20 10-02-30](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/9535138a-3135-45d3-a07d-a885c3a22f9a)

Screenshots
![Screenshot from 2024-02-20 10-02-30](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/faa16600-f838-4f15-b35f-7a57ba8cb970)

![Screenshot from 2024-02-20 10-03-22](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/0e5be1d7-54c0-43a0-98d6-09141719d482)

![Screenshot from 2024-02-20 10-03-07](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/1bf9d713-a8b2-4802-ac97-dfed0477e44b)

![Screenshot from 2024-02-20 10-03-29](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/08385708-3000-4400-9788-1a19801911d9)

![Screenshot from 2024-02-20 10-03-34](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/ff00bce7-c336-4bad-8655-19d61d4ad074)

![Screenshot from 2024-02-20 10-03-39](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/35eb5c42-0811-47ab-a38c-8442677ae099)

![Screenshot from 2024-02-20 10-03-45](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/ac58f91a-9335-4a0d-af9e-742e248bf288)

![Screenshot from 2024-02-20 10-03-49](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/be8bfee2-228f-47a5-a799-698ef1c5b764)

![Screenshot from 2024-02-20 10-03-54](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/01b63130-0d67-4769-92d2-80f4d98b5e56)

![Screenshot from 2024-02-20 10-03-58](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/18245881-9a72-4cac-89fd-b2784be83d6a)

![Screenshot from 2024-02-20 10-04-02](https://github.com/Muthomi254/TravelApp-Backend/assets/140247195/e5f95277-0751-4330-b9cc-3f7ee3b6281b)
